app/controllers/autoscaling_controller.rb

```diff
@@ -0,0 +1,5 @@
+class AutoscalingController < ApplicationController
```

Comment:
This name feels a little specific to me - health checks are useful for autoscaling, but also other things (alerts, etc). I'd be inclined to try and reference the user in the module names (and maybe the path too?). I guess the user is `monitoring`, or something thereabouts.

I'm not sure how you've done naming/grouping for controllers in your apps so far, seems like a good opportunity to ask!

This isn't serious enough to stop you from merging.

Reply:
Agreed.